client/tracing/src/lib.rs

```diff
@@ -440,12 +440,11 @@ mod tests {
 		}
 	}
 
-	type TestSubscriber = tracing_subscriber::layer::Layered<
-		ProfilingLayer,
-		tracing_subscriber::fmt::Subscriber
-	>;
-
-	fn setup_subscriber() -> (TestSubscriber, Arc<Mutex<Vec<SpanDatum>>>, Arc<Mutex<Vec<TraceEvent>>>) {
+	fn setup_subscriber() -> (
```

Comment:
I don't understand why I see here all my changes again.

Reply:
I had cherry-picked stuff. I just need to update the branch again